internal/authn/oidc/oidc.go

```diff
@@ -21,13 +21,15 @@ import (
 	"google.golang.org/grpc/status"
 
 	"github.com/openfga/openfga/internal/authn"
+	authnPkg "github.com/openfga/openfga/pkg/authn"
```

Comment:
Can we rename this package to something like `authclaims`, the `authnPkg` name feels a bit verbose.